microprofile-fault-tolerance/pom.xml

```diff
@@ -52,6 +52,13 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
```

Comment:
you don't need the whole dependencyManagement section here as both boms are in parent.

Reply:
That's done towards the efforts of removing the boms from parent as they should not be there.

Reply:
Resolving discussion - cc @emmartins